docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -0,0 +1,460 @@
+
+.. _prereqs-multi-ubuntu-label:
+
+Prerequisites
+-------------
+
+This procedure requires a working (runnable) validator node with a validator,
+REST API, and the Settings, IntegerKey, and XO transaction processors
+(see :doc:`ubuntu`). This procedure describes how to start the XO transaction
+processor if it is not already running.
+
+For each validator node that will be on your network, gather the following
+information:
+
+* Where this validator will listen for incoming communication from this
+  validator's components (the "component bind string")
+
+* Where this validator will listen for incoming communication from other
+  validator nodes, also called peers (the "network bind string")
+
+* The address that other peers should use to find this validator node (the
+  "public endpoint string")
+
+* The addresses that this validator should use to connect to the other validator
+  nodes, also referred to as peers (the "peers list"). Each address on the list
+  is referred to as the "peer's public endpoint string".
+
+.. _about-bind-strings-label:
+
+About component and network bind strings
+++++++++++++++++++++++++++++++++++++++++
+
+For the network bind string and component bind string, you would typically use
+a specific network interface that you want to bind to.
+The ``ifconfig`` command provides an easy way to determine what this interface
+should be. ``ifconfig`` displays the network interfaces on your host system,
+along with additional information about the interfaces. For example:
+
+.. code-block:: console
+
+   $ ifconfig
+   eth0      Link encap:Ethernet  HWaddr ...
+             inet addr:...  Bcast:...  Mask:255.255.0.0
+             UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
+             RX packets:17964 errors:0 dropped:0 overruns:0 frame:0
+             TX packets:6134 errors:0 dropped:0 overruns:0 carrier:0
+             collisions:0 txqueuelen:0
+             RX bytes:26335425 (26.3 MB)  TX bytes:338394 (338.3 KB)
+   lo        Link encap:Local Loopback
+             inet addr:127.0.0.1  Mask:255.0.0.0
+             UP LOOPBACK RUNNING  MTU:65536  Metric:1
+             RX packets:0 errors:0 dropped:0 overruns:0 frame:0
+             TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
+             collisions:0 txqueuelen:1
+             RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
+
+This example output shows that ``eth0`` is a network interface that has access
+to the Internet. In this case, you could use one of the following:
+
+* If you would like the validator node to accept connections from other
+  validator nodes on the network behind ``eth0``, you could specify a
+  network bind string such as ``tcp://eth0:8800``.
+
+* If you would like the validator node to accept only connections from local
+  Sawtooth components locally, you could specify the component bind string
+  ``tcp://lo:4004``. Note that this is equivalent to ``tcp://127.0.0.1:4004``.
```

Comment:
The above sentence is awkward. I suggest, "If you would like the validator node to only accept local connections from Sawtooth components, you could specify the component bind string"

Reply:
Ditto (will fix in a separate PR soon).